sapi/cli/php_cli_server.c

```diff
@@ -1437,14 +1437,13 @@ static void php_cli_server_request_dtor(php_cli_server_request *req) /* {{{ */
 	}
 } /* }}} */
 
-static void php_cli_server_request_translate_vpath(php_cli_server_request *request, const char *document_root, size_t document_root_len) /* {{{ */
+static void php_cli_server_request_translate_vpath(php_cli_server *server, php_cli_server_request *request, const char *document_root, size_t document_root_len) /* {{{ */
```

Comment:
```suggestion
static void php_cli_server_request_translate_vpath(const php_cli_server *server, php_cli_server_request *request, const char *document_root, size_t document_root_len) /* {{{ */
```
Make it const while at it?

Reply:
Sure, I should get into the habit of making pointer params `const`. :) If only this was the default. ^^

Reply:
Yeah, there is also the difficulty that sometimes one cannot use it because an underlying API doesn't mark the params as const properly :/